hack/ensure-kind.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Copyright 2021 The Kubernetes Authors.
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
why going back in years here?

Reply:
CAPI script has 2019 as the copyright, despite having newer changes than us. I took their script as is and then added the path check + warning on top. Which year you prefer to have there, I never understood the copyright years that are not actively maintained along with changes, so to me both are equally bad.

Reply:
No strong opinion, from my experience these scripts are merged once (with proper year) and not touched that frequently. But when touched, years can be updated to reflect the current date. 

/approve